styles.css

```diff
@@ -60,7 +60,65 @@ ul {
   display: none;
 }
 
+.close {
+  display: none;
+}
+
+.nav-links {
+  display: none;
+}
+
 @media (max-width: 767px) {
+  .visible {
+    position: fixed;
+    height: 95vh;
```

Comment:
- [ ] One of the requirements in this milestone is to follow the Figma design. You have done a good job in styling but a little problem is that as you are giving it `height: 95vh`, the background of the popup menu bar is not covering 100% like in the figma design, so kindly consider changing it to `height: 100vh` to get the desired results.
your design

![re](https://user-images.githubusercontent.com/96683843/231265678-0ab91ce1-b15e-4154-af27-5f4531ff083f.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/231265651-872ffda7-bfbf-451c-9664-3d0ca1b1418c.jpg)